src/common/types.ts

```diff
@@ -1,29 +1,33 @@
+import Club from '../entities/Club';
+import Interest from '../entities/Interest';
+import CornellMajor from '../entities/CornellMajor';
+
 //** Represents a User */
 export interface SerializedUser {
   clubs: SerializedClub[];
   firstName: string;
   googleID: string;
-  graduationYear: string;
-  hometown: string;
+  graduationYear: string | null;
```

Comment:
We used both undefined and null, can we standardize to just using one?

Reply:
From my understanding, we can't. [TypeORM doesn't treat undefined the same way it does null](https://github.com/typeorm/typeorm/issues/2934), and when I try to use null in the `UserUpdateFields` interface instead of undefined I get the error `Type 'null' is not assignable to type 'DeepPartial<Club>[] | undefined'.` which comes from the typeorm `merge` function.

Reply:
Can we use undefined for these guys then?

Reply:
I don't think so: `undefined` doesn't work the same way `null` does in TypeORM, in that even if you have an entity's field as nullable and the field type as optional (i.e. `<fieldType> | undefined`) then setting that field to undefined and saving the entity to the database does nothing, the field will stay the same. For example, if the field `graduationYear` in the user entity class is nullable and is currently set to `2022` and then we do `user.graduationYear = undefined`, the value will still be `2022` even after calling `db().save(user)`. Using null gives the expected behavior of setting the value of that field to `null` in the db. 
Let me know if that makes sense or if you think I am missing something here.

Reply:
Makes sense, thanks for the info!